### DISCUSSION TITLE
Is it advised to use netavark instead of CNI or is still too soon?

### DISCUSSION THREAD

**Discussion by Folaht** [containers/podman#13474] [Q&A] [answered]
And what's the best tutorial on understanding and setting up netavark?

I've only found https://podman.io/blogs/2022/02/04/network-usage.html and that's for Fedora.
I use Manjaro.

**Comment by Luap99** [comment, marked as answer]
Netavark is the default for podman 4.0 when it is installed and it was not an upgrade from 3.X.
So yes please use netavark whenever possible. The only think still missing in netavark is dhcp support. Also only bridge and macvlan configs are supported, so if you have custom cni plugins this will not work with netavark obviously.

The use of netavark improves dns functionality massively with aardvark-dns.

To use netavark you should install it and aardvark-dns into one of these directories: https://github.com/containers/common/blob/6fdd46f5ab219830d3b85f0c2bff5b350630d84c/pkg/config/config_linux.go#L39-L44
Then run `podman system reset`. Now podman should default to netavark. You can check this with `podman info --format {{.Host.NetworkBackend}}`
